### PR TITLE
Verify Account Through Link

### LINKED ISSUE #2018: Verify Account Through Link
**Is your feature request related to a problem? Please describe.**
With the verify account email that is sent through the `sendEmailVerification` mutation, and soon to be sent during sign up, the link that is received through an email does not work with the frontend.

**Describe the solution you'd like**
What should happen is when the user clicks on the link from the email, it should open tracker to the `/validate/<token>`. The frontend should take the token and automatically run the `verifyAccount` mutation with the token from the URL.

**Additional context**
I was thinking how this should look, and I like the loading animation for pages where its just the loading ciricle and info telling the user what its loading. I think something along those lines would look great.

### PR DESCRIPTION
**Pull request by lcampbell2**
Resolves #2018 

Adds in page (`/validate/:verifyToken`) linked from verification emails that uses `verifyAccount` mutation to do as such for users

Screenshot from 2021-05-05 14-09-21

`Verify Email` button added to User Page that manually sends email containing link if account has not been verified

Screenshot from 2021-05-05 15-54-48

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { Page } from './Page'
 
 const PageNotFound = lazy(() => import('./PageNotFound'))
 const CreateUserPage = lazy(() => import('./CreateUserPage'))
-const QRcodePage = lazy(() => import('./QRcodePage'))
+// const QRcodePage = lazy(() => import('./QRcodePage'))
```
Comment (nsdeschenes) [inline frontend/src/App.js:23]: This can probably just be straight up removed, I don't believe we are going to use the QR code again.

**Review by nsdeschenes — CHANGES_REQUESTED** [review]

File: frontend/src/EmailValidationPage.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    verifyAccount({ variables: { verifyToken: verifyToken } })
+  }, [verifyAccount, verifyToken])
+
+  const verifyMessage = () => {
+    if (success) {
+      return (
+        <Stack isInline align="center">
+          <Icon name="check-circle" color="strong" />
+          <Text fontSize="xl">
+            <Trans>Your account email was successfully verified</Trans>
```
Comment (nsdeschenes) [inline frontend/src/EmailValidationPage.js:37]: Can a `home` button be put here, just to give a super duper clear way for what the user needs to do next?

File: frontend/src/EmailValidationPage.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          <Stack isInline align="center">
+            <Icon name="warning" color="weak" />
+            <Text fontSize="xl">
+              <Trans>
+                Your account email could not be verified at this time. Please
+                try again.
+              </Trans>
+            </Text>
+          </Stack>
+          <Text fontSize="xl" textAlign="center">
```
Comment (nsdeschenes) [inline frontend/src/EmailValidationPage.js:53]: And maybe a `home` button here as well, something to give the user something extremely clear on where to go next.

**Review by nsdeschenes — CHANGES_REQUESTED** [review]

**Review by nsdeschenes — CHANGES_REQUESTED** [review]

**Review by nsdeschenes — APPROVED** [review]

**Review by FestiveKyle — APPROVED** [review]

**Pull request merged**